### ISSUE TITLE
Completion is incorrect for lambda parameters

### ISSUE THREAD

**Issue by 333fred** [dotnet/roslyn#35222]
```C#
using System;

namespace ConsoleApp2
{
    class Program
    {
        static void Main(string[] args)
        {
            Test(prog => prog.[||]); // Invoke completion on prog.
        }

        static void Test(Action<Program> d)
        {
        }
    }
}
```

Invoking completion here brings up the members of `Action<Program>`, not the members of `Program`, rendering completion completely useless. cc @ivanbasov.

**Comment by 333fred** [comment]
Likely a dup of https://github.com/dotnet/roslyn/issues/35027

**Comment by ivanbasov** [comment]
Should be fixed with https://github.com/dotnet/roslyn/pull/35067 but I will double check.

**Comment by ivanbasov** [comment]
Verified. Fixed with #35067
![image](https://user-images.githubusercontent.com/5455484/56617953-0f8fcd80-65d6-11e9-9afe-93c3d38b1492.png)
